analyses/malawi/dryspells_trigger/01_chirps_dry_spell_detection.R

```diff
@@ -78,14 +79,18 @@ s2017 <- raster::stack(paste0(chirps_path, "chirps_global_daily_2017_p05.nc"))
 s2018 <- raster::stack(paste0(chirps_path, "chirps_global_daily_2018_p05.nc"))
 s2019 <- raster::stack(paste0(chirps_path, "chirps_global_daily_2019_p05.nc"))
 s2020 <- raster::stack(paste0(chirps_path, "chirps_global_daily_2020_p05.nc"))
+s2021 <- raster::stack(paste0(chirps_path, "chirps_global_daily_2021_p05.nc"))
 
```

Comment:
I had originally done something very similar. It took a few hours or days to realise it caused issue because it obfuscated the column names. There may be a way to read them recursively and keep the column names (which are the dates, so a very important piece of information), but I couldn't find it at the time and had spent too much time on it while we were on a time crunch. That's why I had left a comment in the code saying it HAD to be done manually or the col names would be lost. 
Agreed that it's not worth looking into if refactoring in Py.

Reply:
Yes sorry that was my mistake..